extensions/AbuseFilter/AbuseFilter.hooks.php

```diff
@@ -100,7 +100,9 @@ public static function onAbortMove( $oldTitle, $newTitle, $user, &$error, $reaso
 		return $filter_result == '' || $filter_result === true;
 	}
 
-	public static function onArticleDelete( &$article, &$user, &$reason, &$error ) {
+	public static function onArticleDelete(
+		WikiPage $article, User $user, $reason, &$error
+	): bool {
```

Comment:
Thanks @Krinkle ! Yeah, my rationale was that adding the explicit return type hint can be useful if a code path in the handler can result in early abort, while it does not do any harm at all if we always allow processing to continue.